### PR TITLE
[1.x] Internationalization support

### PR DESCRIPTION
**Pull request by KanekiYuto**
I provide internationalization support, and you just need to execute the following command to publish the language file.
Currently only support English, Chinese (Simplified), Chinese (Traditional)
````
php artisan pulse:lang-publish
````

### CODE REVIEW

File: resources/views/components/card-header.blade.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
                     {{ $icon }}
                 </div>
             @endif
-            <hgroup class="flex flex-wrap items-baseline gap-x-2 overflow-hidden">
+            <hgroup class="flex flex-nowrap items-baseline gap-x-2 overflow-hidden">
```
Comment (timacdonald) [inline resources/views/components/card-header.blade.php:10]: Why did you make this change?

**Comment by taylorotwell** [comment]
Hey there! We don't plan to provide official internationalization support for Pulse at this time. It's just a bit too much for us to maintain. Thank you.

**Comment by KanekiYuto** [comment]
> Hey there! We don't plan to provide official internationalization support for Pulse at this time. It's just a bit too much for us to maintain. Thank you.
I think the internationalization work can also be done by the community, and the main maintainer only needs to provide the most basic English language pack

**Comment by KanekiYuto** [comment]
I hope you can accept my offer. Thank you
@taylorotwell

**Pull request closed without merging**